Initial draft

greg0ire opened the pull request.

Commit:
Briefly summarize project

Commit:
Create Symfony application

Commit:
Dockerize project

Commit:
Generate film manifest

Commit:
Make the program usable without docker-compose

In production, we plan on using it with docker and environment
variables.

Commit:
Document usage

L-P commented on Makefile:
```diff
@@ -0,0 +1,10 @@
+
+# To be run from the host
+.env:
+	@ cp $@.dist $@
```
-@

L-P commented on composer.json:
```diff
@@ -0,0 +1,31 @@
+            "DigitalBackstage\\OrangeJuicer\\": "src"
+        }
+    },
+    "license": "proprietary",
```
Nope.

L-P commented on docker/docker-compose.override.yml:
```diff
@@ -0,0 +1,10 @@
+  orange_juicer:
+    entrypoint: docker/dev-entrypoint.sh
+    volumes:
+      - ~/Vidéos:/data
```
Specific to your setup

L-P commented on spec/ManifestGeneratorSpec.php:
```diff
@@ -0,0 +1,50 @@
+        $this->beConstructedWith(
+            $encoder,
+            $filesystem,
+            array()
```
PHP 5.4 is long gone, `[]`

L-P commented on src/ManifestGenerator.php:
```diff
@@ -0,0 +1,74 @@
+    ) {
+        $metadata = [];
+        $metadata['title_1'] = $title;
+        $metadata['multi_format_set']
```
Confusing indentation

greg0ire commented on src/MetadataProvider/MediaInfoProvider.php:
```diff
@@ -0,0 +1,76 @@
+
+        list($width, $height) = explode(':', $aspectRatio);
+        /* Remove cast to int when
+         * https://github.com/mhor/php-mediainfo/issues/65 is fixed */
```
Do not merge yet, this has been fixed and released :)

Commit:
Put Composer lock file under version control

This is not a library.

Commit:
Check for manifest before doing heavy computing

Computing the has then getting an error explaining the file can't be
overwritten would be a waste of time.

Commit:
Use a choice widget for languages

Commit:
Remove extra wrapping tag

Commit:
Play nice with optional suffix

Commit:
Display booleans as strings

Commit:
Add support for trailers

Commit:
Setup CI

L-P commented on src/TrailerDetector.php:
```diff
@@ -0,0 +1,29 @@
+    public function getTrailerFilename(string $programFilename): string
+    {
+        return preg_replace(
+            '|MPEG\d+\.\mpg|',
```
`\m` ?
